### ISSUE TITLE
How to translate the specification page (not the specification documents) on jakarta.ee?

### ISSUE THREAD

**Issue by laliwa** [jakartaee/jakarta.ee#924]
Hi everyone,

Need to decide which is the better way to translate the content (which we see exactly on the page without clicking) here https://jakarta.ee/specifications/ .

Specifications are showing with title and description in blocks, for the appearance, there are two types of them:
type1. the description is short enough to show completely (e.g. Jakarta MVC)
type2. the descriptions are too long and some of them are hidden with "..." by the end

We need to decide how to translate type 2:
solution 1. translate it as it is into a Chinese sentence with "..." at the end
solution 2. leave it not translated 
solution 3. other solutions

will appreciate very much with all your opinions!

**Comment by TanjaObradovic** [comment]
Specification Committee has agreed to have a short 60 character description for all specifications on https://jakarta.ee/specifications/ page. At this point only Jakarta MVC has this implemented. 
This topic in on the agenda for the next Specification Committee call, and I expect it will be decided then how to make progress and implement short description that we can then translate.

**Comment by chrisguindon** [comment]
I created a PR a few days ago to help us get started on this:
https://github.com/jakartaee/specifications/pull/297

The PR synchronize the work that we did so far in order to translate Jakarta.ee pages, in particular, the specification section.

This PR will allow contributors to review their specification changes in all supported languages. The supported languages are  English and Chinese for now.
